pkgs/top-level/python-packages.nix

```diff
@@ -10119,11 +10119,11 @@ in {
 
   graphviz = buildPythonPackage rec {
```

Comment:
This package needs to be moved out of `python-packages.nix`.

Reply:
What are the proper location for this expression?

Reply:
(deleted question about the location. going to add under development/python-modules)